ClinGen/clincoded
# Connects to #1150. Interpretation list

![image](https://cloud.githubusercontent.com/assets/4326866/20984700/3a6503ba-bc76-11e6-98e3-458288dbcb55.png)

Adds master interpretation list, much like the master GDM list. Adds link to the master interpretation list to Dashboard. Minor changes to master GDM list for consistency in sorting and display. Changes to JSON schema files is to add additional return fields, not actual schema changes. 'Last Edit' column value is only available/dependent on Evaluations in Interpretation.

Testing:

1. Go to Dashboard
2. Click on 'View all interpretations' link
3. Confirm that columns sort properly and that the filtering works fine
4. Confirm that clicking on interpretation row takes you to that variant in VCI
